packages/website/src/components/editor/LoadedEditor.tsx

```diff
@@ -28,12 +28,25 @@ export interface LoadedEditorProps extends CommonEditorProps {
   readonly webLinter: WebLinter;
 }
 
+export function determineLanguage(file: string): string {
+  if (/\.[mc]?ts(x)?$/.test(file)) {
+    return 'typescript';
+  }
+  if (/\.[mc]?js(x)?$/.test(file)) {
+    return 'javascript';
+  }
+  if (/\.(json|eslintrc)$/.test(file)) {
+    return 'json';
+  }
+  return 'plaintext';
+}
```

Comment:
could this use this function instead?

https://github.com/typescript-eslint/typescript-eslint/blob/399b3b1263665288316be6faff759cf514e24b1b/packages/typescript-estree/src/create-program/getScriptKind.ts#L4-L34

Reply:
in theory we could use it in here

```
export function determineLanguage(file: string): string {
 const kind = getScriptKind()
 switch (extension) { 
   case window.ts.ScriptKind.JS:
   
 }
}
```

but we still need extra check for `eslintrc`

Reply:
after further considiration, i actually don't need it i can just let ts decide what language to choose base on available extensions